pallets/subtensor/rpc/src/lib.rs

```diff
@@ -46,6 +46,10 @@ pub trait SubtensorCustomApi<BlockHash> {
     fn get_subnet_info(&self, netuid: u16, at: Option<BlockHash>) -> RpcResult<Vec<u8>>;
     #[method(name = "subnetInfo_getSubnetsInfo")]
     fn get_subnets_info(&self, at: Option<BlockHash>) -> RpcResult<Vec<u8>>;
+    #[method(name = "subnetInfo_getSubnetInfo_v2")]
+    fn get_subnet_info_v2(&self, netuid: u16, at: Option<BlockHash>) -> RpcResult<Vec<u8>>;
+    #[method(name = "subnetInfo_getSubnetsInf_v2")]
+    fn get_subnets_info_v2(&self, at: Option<BlockHash>) -> RpcResult<Vec<u8>>;
```

Comment:
Why do all our custom rpc return bytes? It makes them annoying to consume, and returning a SCALE encoded type is no less efficient.

Reply:
yeah 100% agree on this, opens us up to all sorts of potential footguns and even vulnerabilities potentially on the input side if certain criteria are met

Reply:
> what would you rather they returned ?

The actual shape of the data, `SubnetInfov2`/`Vec<SubnetInfov2>`/`Result<Option<SubnetInfov2>>` / whatever it is before you call `.encode`